app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,8 @@
 class PostsController < ApplicationController
   before_action :set_post, only: [:show, :edit, :update, :destroy]
 
+  around_filter :hypernova_render_support
```

Comment:
Change to application controller will cause unnecessary server side rendering (ex. admin dashboard). 🍣